src/oblogger_view.ts

```diff
@@ -794,12 +847,17 @@ export class ObloggerView extends ItemView {
         // Dump whatever remains (hopefully not much)
         this.otcGroupsDiv?.empty();
 
+        // todo: need to add a separator somewhere in here?
```

Comment:
@robotbrad do we want to figure out UI for separating property groups from tag groups in this PR or in a follow up PR?

Reply:
expanded in #134